waltz-tools/src/main/java/com/wepay/waltz/tools/storage/StorageCli.java

```diff
@@ -156,21 +206,24 @@ private String getMetricsJson(String storageHost, int storagePort, String cliCon
             return partitionInfo;
         }
 
-        private void listPartitionInfo(Map<Integer, PartitionInfoSnapshot> partitionInfo) {
+        private StringBuilder formatPartitionInfo(Map<Integer, PartitionInfoSnapshot> partitionInfo, String storageHost, String storagePort) {
             StringBuilder sb = new StringBuilder();
-
             // display partition info
             for (Map.Entry<Integer, PartitionInfoSnapshot> entry: partitionInfo.entrySet()) {
                 int partitionId = entry.getKey();
                 PartitionInfoSnapshot snapshot = entry.getValue();
-                sb.append(String.format("Partition Info for id: %d%n", partitionId));
+                sb.append(String.format("Partition Info for id: %d at %s:%s%n", partitionId, storageHost, storagePort));
                 sb.append(String.format("\t sessionId: %d%n", snapshot.sessionId));
                 sb.append(String.format("\t lowWaterMark: %d%n", snapshot.lowWaterMark));
                 sb.append(String.format("\t localLowWaterMark: %d%n", snapshot.localLowWaterMark));
                 sb.append(String.format("\t isAssigned: %b%n", snapshot.isAssigned));
                 sb.append(String.format("\t isAvailable: %b%n", snapshot.isAvailable));
             }
-            System.out.println(sb.toString());
+            return sb;
```

Comment:
nit: why not return `sb.toString()` 😃

Reply:
To minimize the frequency of conversions to a string. While inconsequentially faster, I thought it just made for better design.

Reply:
okay